TestMain.py

```diff
@@ -73,24 +77,27 @@ def test_build_specimin_command(self):
                    }]
         specimin_dir = 'user/specimin'
         target_dir = 'user/ISSUES/cf-6019'
-        command = main.build_specimin_command(proj_name, target_dir, specimin_dir, root, targets)
+        command = main.build_specimin_command(proj_name, target_dir, root, targets)
         with open('resources/specimin_command_cf-6019.txt','r') as file:
             target_command = file.read()
         self.assertEqual(command, target_command)
         #not executing since it crashes specimin.
 
         # make 
         issue_name = self.json_data[JsonKeys.ISSUE_ID.value]
-        main.create_issue_directory('resources', issue_name)
-        self.assertTrue(os.path.exists('resources/cf-1291/input'))
-        main.clone_repository(self.json_data[JsonKeys.URL.value], f"resources/{issue_name}/input")  
+        resource_abs_path = os.path.abspath("resources")
+        issue_input_dir = main.create_issue_directory(resource_abs_path, issue_name)
+
+
+        self.assertTrue(os.path.exists(issue_input_dir))
+        main.clone_repository(self.json_data[JsonKeys.URL.value], issue_input_dir)  
```

Comment:
Why are any of these changes necessary? They seem unrelated to the location of specimin, and only have to do with the location of the issues (which shouldn't need to change to adapt this to use a local copy of specimin).

Reply:
Professor, I have removed any unnecessary changes. Any refactoring will be added as a separate commit later.